src/ScottPlot5/ScottPlot5/Primitives/Label.cs

```diff
@@ -30,57 +30,11 @@ public class Label
     public PixelOffset ShadowOffset { get; set; } = new(3, 3);
 
     public bool AntiAliasBackground { get; set; } = true;
-    public bool AntiAliasText { get; set; } = true;
-
-    // TODO: use a class for cached typeface management
-
-    private SKTypeface? CachedTypeface = null;
-    private SKTypeface Typeface
-    {
-        get
-        {
-            if (CachedTypeface is not null)
-                return CachedTypeface;
-
-            if (FontFile is not null)
-                return FontStyle.CreateTypefaceFromFile(FontFile);
-
-            return FontStyle.CreateTypefaceFromName(FontName, Bold, Italic);
-        }
-    }
-
-    public void SetTypeface(SKTypeface typeface)
-    {
-        CachedTypeface = typeface;
-    }
-
-    private string _FontName = Fonts.Default;
-    public string FontName
-    {
-        get => _FontName;
-        set { _FontName = value; ClearCachedTypeface(); }
-    }
-
-    private string? _FontFile = null;
-    public string? FontFile
```

Comment:
We need to preserve this for backwards compatibility.

Reply:
Already answered a little higher.